docs/xxoriginaljadk8heapsizecompatibilitymode.md

```diff
@@ -0,0 +1,40 @@
+<!--
+* Copyright (c) 2020, 2020 IBM Corp. and others
+*
+* This program and the accompanying materials are made
+* available under the terms of the Eclipse Public License 2.0
+* which accompanies this distribution and is available at
+* https://www.eclipse.org/legal/epl-2.0/ or the Apache
+* License, Version 2.0 which accompanies this distribution and
+* is available at https://www.apache.org/licenses/LICENSE-2.0.
+*
+* This Source Code may also be made available under the
+* following Secondary Licenses when the conditions for such
+* availability set forth in the Eclipse Public License, v. 2.0
+* are satisfied: GNU General Public License, version 2 with
+* the GNU Classpath Exception [1] and GNU General Public
+* License, version 2 with the OpenJDK Assembly Exception [2].
+*
+* [1] https://www.gnu.org/software/classpath/license.html
+* [2] http://openjdk.java.net/legal/assembly-exception.html
+*
+* SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH
+* Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
+-->
+
+# -XX:\[+|-\]OriginalJDK8HeapSizeCompatibilityMode
+
+![Start of content that applies only to Java 8 (LTS)](cr/java8.png) When this option is enabled, revert the default value of Maximum Heap Size to the behavior in OpenJ9 0.18 and earlier releases(). 
+
+<i class="fa fa-exclamation-triangle" aria-hidden="true"></i> **Note:** This option is deprecated in all versions.
```

Comment:
If this option is for Java 8 only, why would we say it was deprecated in all versions or am I misunderstanding?

Reply:
the option is only for Java8 and ignored on Java 11+, set the option to deprecated, because we do not recommend the users to use it, but in case they want to match old behavior, then they can use it.

Reply:
OK, So I don't think we need to say "in all versions"